tests/electron/env_spec.js

```diff
@@ -0,0 +1,40 @@
+const helpers = require("./global-setup");
```

Comment:
Here a header comment with license is missing

Reply:
this is missing in the most test files, I did split this file but the original also didn't contain a header comment.